File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    print(ord("a"), ord("z"), ord("A"), ord("Z"), ord(" "))
    for i in s:
        ascii_i = ord(i)
        if ascii_i == 32:
            answer += i
        elif 65 <= ascii_i <= 90:
            answer += chr(ascii_i + n) if ascii_i + n <= 90 else chr(ascii_i + n - 26)
        elif 97 <= ascii_i <= 122:
            answer += chr(ascii_i + n) if ascii_i + n <= 122 else chr(ascii_i + n - 26)
            
    return answer  

File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    var answer = [...s].map((el) => {
        if (el === " ") return el
        
        const isUpperCase = el.toUpperCase() === el
        const code = el.toLowerCase().charCodeAt() + n
        const incode = String.fromCharCode(code > 122 ? code - 26 : code)
        return isUpperCase ? incode.toUpperCase() : incode
    })
    return answer.join("");
}